content/english/anti-vegan/no-food-is-fully-vegan.md

```diff
@@ -11,33 +11,103 @@ weight: 5
+
+> Argument #1 is a fallacy (the appeal to hypocrisy).
+
+### Argument #2 *Intent & Knowledge*
+
+Vegans must eat and, where there are limited options, follow the path of least harm.
+
+
+>A golden rule of moral philosophy is, "all things being equal you ought to choose the action that does the least harm”.
+
+While it is true that some animals do die in the production and harvesting of crops vegans do not ‘accept these deaths’ without question.  Where a crop farmer fails to *exercise due diligence to minimize the harm* and deaths occur, an obligation falls on vegans *as far as is possible and practicable* to seek alternatives (e.g. moving towards organic vegetables, buying from [vegan farms] (https://www.viva.org.uk/what-we-do/vegan-farming), avoiding mono-cultures etc.)
+
+> ### Doctrine of Double Effect
+> - the nature of the act is itself good, or at least morally neutral;  
+> - the individual intends the good effect and does not intend the bad effect, either as a means to the good or as an end in itself;  
+> - the good effect outweighs the bad effect in circumstances sufficiently grave to justify causing the bad effect, and the individual exercises due diligence to minimize the harm.  
+> 
+> *from* Double-Effect Reasoning: Doing Good and Avoiding Evil *by* T. A. Cavanaugh.
+ 
+In other words a cattle farmer *intends* to kill the cattle, the crop farmer's intention is primarily to grow crops. Any deaths that may occur during crop harvest are a side-effect of a morally neutral act. 
+
+- A counter might follow that 'regardless of intention or acceptance the animals die, so I might as well get my meat.'
+
+This counter argument fails to take into account an individual's foreknowledge of what will occur.
 
-Ed Winters (aka Earthling Ed) hits the nail on the head with zir comparison in zir book '30 Non Vegan Excuses'.
+* A cattle farmer knows that the cattle will be slaughtered and intends for them to be slaughtered.
 
-> If someone was driving their car and they accidentally hit a dog, that would not be the same as if they purposefully drove after the dog until they ran them over.  
-> - Ed Winters - 30 Non Vegan Excuses - p72
+* A crop farmer knows that some animals (e.g rabbits, birds, rodents) may be killed during harvest but the intention is only to harvest crops. (The crop farmer would far prefer the animals were not in the field).
+ 
+Consider the following thought experiment:
 
-Intention matters in the morality of your actions. The unintentional deaths of earthworms when tilling, or field mice when harvesting are not _necessary_ for the production of crops, it is not the intention to kill animals when harvesting crops. It is the definitionally the intention when raising animals for slaughter.
+> You are controlling a tram. As long as you hold a lever the tram will continue on its current track.  However, you know there are numerous people on the track ahead who will be killed if the tram continues.  
```

Comment:
Can we remove some of the qualifiers on this? Specifically:
- Numerous
- Very probably

So we would get 

>  You are controlling a tram. As long as you hold a lever the tram will continue on its current track.  However, you *know* there are people on the track ahead who will be killed if the tram continues.  
> If you release the lever the tram will change tracks. You *suspect* there are people on the other track as well.

It conveys the same uncertainty but makes it easier to discern the difference.